src/main/resources/db/migration/V209__alter_column_epim_id.sql

```diff
@@ -0,0 +1,29 @@
+-- FACT-1903
+-- drop constraint on epim id
+-- change epim id to varchar(30)
+-- readd the constraint to ensure epim id length is less than or equal to 30
+DO $$
```

Comment:
Could you explain how this works? Looks very GPT generated

Reply:
initially i had just add the column if its not there but it was a length of 255

then i accidently merged, i reverted it

then i edited 208 to add the constraint forgetting i had merged it already. (i changed it back as requested now)

then added 209 thinking the constraint would be in before running but it won't run again so the constraint won't be there.

209 will run and remove the constraint if it exists (which it shouldn't exist anyway so i could remove it) otherwise  it'll run the risk of having a constraint allowing more than the length of column has space for.

then it'll change the length from 255 to 30 using a left which just clips off anything after a length of 30 because we'd need to tell it how to convert the data (even though there shouldn't be anything there anway)

then i re add back in the constraint of length up to 30.

i used 'Do begin' with 'if' statements because i tend to try use use 'if exists' and 'if not exists' a lot and not all queries allow you to use it straight in the same line